csrc/pybind.cpp

```diff
@@ -82,6 +82,15 @@ PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
     &moe_align_block_size,
     "Aligning the number of tokens to be processed by each expert such that it is divisible by the block size.");
 
+  ops.def(
+    "quant_per_tensor",
```

Comment:
I'm afraid that this op may confuse people because its name is a bit too general. Given that we already have `scaled_fp8_quant`, can we have a similar name to build a good naming convention?

Reply:
`scaled_int8_quant` would make sense

Reply:
Renamed to `static_scaled_int8_quant` to match the fp8 interface closely.